main.go

```diff
@@ -72,9 +72,17 @@ func getHostname() string {
 	return hostname
 }
 
+func heartbeat() {
+	for _ = range time.Tick(1 * time.Second) {
```

Comment:
@burnsed not sure how often heka used to log a heartbeat, so I just made it every second...